ivy/functional/backends/jax/device.py

```diff
@@ -62,7 +64,8 @@ def dev_to_str(device):
     return p + ":" + str(dev_id)
 
 
-def dev_from_str(device):
+def dev_from_str(device: jax.String)\
```

Comment:
please reformat this to match the style, you can refer to the other completed ones. Here is an example:

```
def dev_from_str(
       device: jax.String
) -> jax.Device:
...
...
...
```